DigitalOffice/Services/ProjectService/Mappers/DbProjectMapper.cs

```diff
@@ -11,7 +11,7 @@ public DbProject Map(NewProjectRequest request)
         {
             if (request == null)
             {
-                throw new ArgumentNullException("Request is null");
+                throw new ArgumentNullException(nameof(request));
```

Comment:
Why are the changes?

Reply:
It's bad practice to go into someone else's code. But since my project covers all services, I decided to fix all the warnings. It is more correct to use nameof.

Reply:
Why is this bad? Then already do so: `ArgumentNullException($"{nameof(request)} cannot be null")`.

Reply:
For what purpose? The error type already carries this information.

Reply:
Ok